include/courtroom.h

```diff
@@ -252,11 +252,20 @@ class Courtroom : public QMainWindow
   void doScreenShake();
   void doRealization();
 
-  // Attempts to save the IC log to a file, and returns a code based on whether or not it was successful
-  // 0 = success, 1 = directory error, 2 = file error
-  int save_ic_chatlog();
+  enum save_log_result {
+    ERR_UNKNOWN, 
+    ERR_CREATEDIR, //couldn't create directory
+    ERR_CREATELOG, //couldn't create log file
+    SUCCESS
+  };
+  save_log_result ic_result;
+  save_log_result ooc_result;
```

Comment:
No globals. If you have to store multiple return values, use a tuple or a struct (though a tuple would be fine in this case; a struct would be kinda overkill).

Reply:
_frantically Googles what a tuple is_